### PR TITLE
Enable Typeform support

### PR DESCRIPTION
**Pull request by romanrizzi**

### CODE REVIEW

File: lib/onebox/engine/typeform_onebox.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    class TypeformOnebox
+      include Engine
+
+      matches_regexp(/^https?:\/\/[a-z0-9]+\.typeform\.com\/to\/[a-zA-Z0-9]+/)
+      always_https
+
+      def to_html
+        typeform_src = build_typeform_src
+
+        <<-HTML
```
Comment (SamSaffron) [inline lib/onebox/engine/typeform_onebox.rb]: Minor... I would use `<<~` here cause this is creating lots of pointless indenting and whitespace now.

Comment (romanrizzi) [reply lib/onebox/engine/typeform_onebox.rb]: Done. Thanks!

File: lib/onebox/engine/typeform_onebox.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        return escaped_src if query_params.has_key?('typeform-embed')
+
+        escaped_src.tap do |src|
+          if query_params.empty?
+            src << '?' unless src.end_with?('?')
+          else
+            src << '&'
+          end
+
+          src << 'typeform-embed=embed-widget'
```
Comment (SamSaffron) [inline lib/onebox/engine/typeform_onebox.rb]: sorry, another minor thing here ... tapping is not really needed here. 

```
 > "a" << "b"
=> 
"ab"
```

So no need to tap escaped_src, it only makes it a bit more confusing and indented.

Comment (romanrizzi) [reply lib/onebox/engine/typeform_onebox.rb]: Good point. I removed the tap call

**Comment by SamSaffron** [comment]
awesome Roman, merged in!

**Comment by dominic-berta** [comment]
@romanrizzi @SamSaffron great work! Though URLs containing dashes or underline are incompatible only due to the triggering regex. Adapting the regex fixed the issue for our client. I will create a PR shortly

**Pull request merged**